webui/react/src/pages/F_ExpList/glide-table/contextMenu.tsx

```diff
@@ -16,9 +16,7 @@ function useOutsideClickHandler(ref: MutableRefObject<any>, handler: (event: Eve
       if (
         ref.current &&
         !ref.current.contains(event.target) &&
-        !(event.target ? (event.target as Element) : null)?.classList?.contains(
-          'ant-dropdown-menu-title-content',
-        )
+        !(event.target as Element).className?.includes('ant-dropdown')
```

Comment:
```suggestion
        event.target instanceof Element &&
        !event.target.className?.includes('ant-dropdown')
```

`target` does not always inherit `Element`, so i think the above code is safer

Reply:
ok I like the use of `instanceof` but have a different version of the condition